core/src/wsv.rs

```diff
@@ -292,20 +292,20 @@ pub struct WorldStateView {
 }
 
 /// Context necessary for deserializing [`WorldStateView`]
```

Comment:
```suggestion
/// Context necessary for deserializing [`Kura`]
```

Reply:
Maybe move to the `kura` module then?

Reply:
@Arjentix we need this for deserializing `WorldStateView`, i renamed it for the sake of consistency. Check out #3800.